docs/DafnyRef/UserGuide.md

```diff
@@ -1159,6 +1158,7 @@ older versions of Dafny.
 * `-noExterns` - ignore `extern` and `dllimport` attributes in the
   program.
 
+<a id="sec-function-syntax"/>
```

Comment:
I think you need a newline after this to go back to markdown (otherwise the whole paragraph below is HTML)